extensions/integration-tests/readme.md

```diff
@@ -6,22 +6,22 @@ extensionInstallers folder: copy the VISX installers for the extensions we would
 src folder: this is where the test file for features should be added, name the file like this: feature.test.ts. e.g. objectExplorer.test.ts
 
 UI automation testing:
-the ADS UI automation test cases should be added under $root/test/smoke/src/sql folder. Each feature should create its own folder and add 2 files, one for accessing the feature and the other for the test cases. For example: objectExplorer.ts and objectExplorer.test.ts. 
+the ADS UI automation test cases should be added under $root/test/smoke/src/sql folder. Each feature should create its own folder and add 2 files, one for accessing the feature and the other for the test cases. For example: objectExplorer.ts and objectExplorer.test.ts. only tested on Windows for now.
```

Comment:
Is this comment only for smoke tests? Seems that we should call out that both smoke and integration test suites are part of our UI automation E2E story